src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/PlacementRequestService.kt

```diff
@@ -49,6 +49,7 @@ class PlacementRequestService(
           mentalHealthSupport = requirements.mentalHealthSupport,
           createdAt = OffsetDateTime.now(),
           application = assessment.application,
+          application = assessment.application as ApprovedPremisesApplicationEntity,
```

Comment:
I'd perhaps pull this out into a variable and throw an exception with a useful message for non-AP applications as seems likely someone in TA will bump into this in the future, e.g.

```
val application = (assessment.application as? ApprovedPremisesApplicationEntity) ?: throw RuntimeException("Only Approved Premises Assessments are currently supported for Placement Requests")
```